### ISSUE TITLE
Don't use legal package names as URLs for special pages under conda-forge/ on anaconda.org

### ISSUE THREAD

**Issue by chrisbarber** [ContinuumIO/anaconda-issues#11750]
`channels` is a popular package on pypi, and it makes sense to use the same name within conda-forge. But the anaconda.org website has a special URL which conflicts.

Is the anaconda.org website on github or no? Some ugly workaround might also be possible without changing the URLs around.

Repro:

go to
https://anaconda.org/conda-forge/channels

Actual:

redirect to login page

Expected:

a page like [this one](https://anaconda.org/conda-forge/daphne)

**Comment by csoja** [comment]
Yes, anaconda.org does have a special landing page for "channels" and it is an unfortunate name conflict.  However, if someone is using conda to attempt to install the channels package from conda-forge, that will work.  (I was able to conda install channels v2.4 noarch on macOS from conda-forge.) Unfortunately, there is no "quick fix" for the website itself.
